LibVLCSharp.Wpf/VLCViewControl.xaml.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace LibVLCSharp.Wpf
+{
+    public partial class VLCViewControl : UserControl, LibVLCSharp.Shared.IVideoView
```

Comment:
To be consistent with the other platforms, can you rename `VLCViewControl` with `VideoView`?

Reply:
Also, I believe your control needs to be `IDisposable` and dispose of libvlc and mediaplayer in the dispose method